src/material.h

```diff
@@ -75,4 +77,13 @@ class material_type
         const mat_burn_data &burn_data( size_t intensity ) const;
 };
 
+namespace materials
```

Comment:
I also agree with his position on YAGNI, it doesn't apply here as his concern is rooted in simply putting the code at the right level rather than adding additional functionality. His example about multiple containers is an illustration of why this is the correct level, not the impetus for the change.